CLI/test/ressources/outputSignature_3/input/Callee.rdz.tcbl

```diff
@@ -0,0 +1,17 @@
+﻿       IDENTIFICATION DIVISION.
+       PROGRAM-ID. Callee.
+       data division.
+       working-storage section.
+       01 VString typedef strict public.
+      %<<
+       Declaration of type
+      %>>
```

Comment:
Could you please add formalized comments above the type definition ?
Multiline comments inside typedef are rare, but formalized comments (or multiline comments) before public typedefs are commonly used.

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/pull/1631/commits/a858b413562c51d3fc725bd16db27729bd73612f